### ISSUE TITLE
Update chromedriver version also for "geordi rspec"

### ISSUE THREAD

**Issue by makmic** [makandra/geordi#191]
I noticed that the `ChromedriverUpdater` code is only being executed for the `geordi cucumber` command but not for `geordi rspec`. I suggest the version check should happen in each case as you can also write RSpec feature specs.

**Comment by codener** [comment]
Good point. However, we need to make sure it does not annoy developers on projects without feature specs.

**Comment by Denionysos** [comment]
Geordi should check per project if the chromedriver Gem is installed, and depending on that do the update

**Comment by makmic** [comment]
Please note that we do not use the chromedriver **gem** but binary: https://makandracards.com/makandra/29465-install-chromedriver-on-linux

**Comment by makmic** [comment]
You could however check for the `selenium-webdriver` gem.

**Comment by tecden** [comment]
The issue was fixed with the release of version 9.5.0
